hack/verify-apimachinery.sh

```diff
@@ -23,7 +23,7 @@ export API_OPTIONS="--verify-only"
 if make apimachinery-codegen; then
 	echo "apimachinery is up to date"
 else
-	echo "\n FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date"
-	echo "\n FAIL: - please run the command 'make apimachinery'"
+	echo -e "\n FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date"
```

Comment:
echo "\n FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date"
```
\n FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date
```
--------------->
echo -e "\n FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date"
```

FAIL: - the verify-apimachinery.sh test failed, apimachinery is not up to date
```